examples/data-sources/powerstore_volumegroup/variables.tf

```diff
@@ -0,0 +1,19 @@
+variable "username" {
+  type=string
+  description = "Stores the username of PowerStore host."
+}
+
+variable "password" {
+  type=string
+  description = "Stores the password of PowerStore host."
+}
+
+variable "timeout" {
+  type=string
+  description = "Stores the timeout of PowerStore host."
+}
+
+variable "endpoint" {
+  type=string
+  description = "Stores the endpoint of PowerStore host. eg: https://10.1.1.1:443 where 443 is port where API requests are getting accepted"
```

Comment:
do we have to include /api/rest in the endpoint?

Reply:
updated description for endpoint